sa/_db-next/migrations/20170718114448_DBCleanup.sql

```diff
@@ -0,0 +1,61 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+DROP TABLE externalCerts;
+
+DROP TABLE identifierData;
+
+ALTER TABLE certificateStatus DROP LockCol;
```

Comment:
You can combine the two DROP columns into a single statement to make it a bit faster. Also can you double check that Boulder won't generate inserts or updates that contain these columns? otherwise replication will fail once the alters are made, and we won't be able to do the role swap.